commands/template_pull_test.go

```diff
@@ -3,15 +3,13 @@
 package commands
 
 import (
-	"bytes"
 	"fmt"
-	"log"
 	"os"
 	"regexp"
-	"strings"
 	"testing"
 )
 
+/*
```

Comment:
@rgee0 If you can take a look I will appreciate that big time. Not sure why tests fail, but when I run the command by hand they pass ..

Reply:
This is the output
```
--- FAIL: Test_templatePull (0.03s)
    --- FAIL: Test_templatePull/ValidRepo (0.00s)
    	/home/martindekov/go/src/github.com/openfaas/faas-cli/commands/template_pull_test.go:27: The directory templates was not created
    	/home/martindekov/go/src/github.com/openfaas/faas-cli/commands/fetch_templates_test.go:70: Directory template was not created: stat template/: no such file or directory
    --- FAIL: Test_templatePull/WithOverwriting (0.00s)
    	/home/martindekov/go/src/github.com/openfaas/faas-cli/commands/template_pull_test.go:46: 
    	/home/martindekov/go/src/github.com/openfaas/faas-cli/commands/fetch_templates_test.go:70: Directory template was not created: stat template/: no such file or directory
```